recipe/install_gstreamer.sh

```diff
@@ -5,6 +5,10 @@
 
 # https://github.com/conda-forge/bison-feedstock/issues/7
 export M4="${BUILD_PREFIX}/bin/m4"
+
+# Maybe do this only for cos7?
+export PKG_CONFIG_PATH=${CONDA_BUILD_SYSROOT}/usr/lib64/pkgconfig
```

Comment:
This shouldn't be needed now. The pkg-config tool was fixed to look in the right spot now.